### PR TITLE
Hello.js

### PR DESCRIPTION
**Pull request by btduong**
@bot say-hello

### CODE REVIEW

**Comment by btduong** [comment]
Another comment

**Comment by btduong** [comment]
Here is another.

**Comment by btduong** [comment]
@bot say-hello

File: src/hello.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("Hello");
```
Comment (btduong) [inline src/hello.js]: @bot hello-world

**Comment by ImranAdan** [comment]
Testing Testing

**Comment by btduong** [comment]
Should reply

File: src/hello.js (hunk 1 above)
Comment (btduong) [inline src/hello.js]: yes

**Comment by btduong** [comment]
Should reply

File: src/hello.js (hunk 1 above)
Comment (btduong) [inline src/hello.js]: wooohooo

Comment (btduong) [inline src/hello.js]: wooohooo

Comment (btduong) [inline src/hello.js]: wooohooo

Comment (btduong) [inline src/hello.js]: wooohooo

**Comment by btduong** [comment]
blahblahblabhla

File: src/hello.js (hunk 1 above)
Comment (btduong) [inline src/hello.js]: @bot say hello

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                      http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+ 
+  <groupId>com.pixel.flipflop</groupId>
+  <artifactId>pxflipflop</artifactId>
+  <version>1.0</version>
+
+  <dependencies>
```
Comment (btduong) [inline pom.xml]: @bot chao

Comment (btduong) [inline pom.xml]: HELLO

File: src/hello.js (hunk 1 above)
Comment (btduong) [inline src/hello.js]: -> 1234567890

Comment (btduong) [inline src/hello.js]: -> 1234567890

File: pom.xml (hunk 2 above)
Comment (btduong) [inline pom.xml]: -> 1234567890

**Pull request closed without merging**